prov/verbs/src/verbs_mr.c

```diff
@@ -138,7 +138,7 @@ vrb_mr_reg_common(struct vrb_mem_desc *md, int vrb_access, const void *buf,
 #if VERBS_HAVE_DMABUF_MR
 	if (flags & FI_MR_DMABUF)
 		md->mr = ibv_reg_dmabuf_mr(md->domain->pd, (uintptr_t) buf,
-					   len, (uintptr_t) buf,
+					   len, (uintptr_t) base_addr,
```

Comment:
Should use 'base_addr + buf (which is offset here)' as iova.

Reply:
Fixed